src/main/java/org/humancellatlas/ingest/export/entity/web/ExportEntityController.java

```diff
@@ -0,0 +1,56 @@
+package org.humancellatlas.ingest.export.entity.web;
+
+import lombok.RequiredArgsConstructor;
+import org.humancellatlas.ingest.core.web.Links;
+import org.humancellatlas.ingest.export.ExportState;
+import org.humancellatlas.ingest.export.entity.ExportEntity;
+import org.humancellatlas.ingest.export.entity.ExportEntityRepository;
+import org.humancellatlas.ingest.export.entity.ExportEntityService;
+import org.humancellatlas.ingest.export.job.ExportJob;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.rest.webmvc.PersistentEntityResource;
+import org.springframework.data.rest.webmvc.PersistentEntityResourceAssembler;
+import org.springframework.data.rest.webmvc.RepositoryRestController;
+import org.springframework.data.web.PagedResourcesAssembler;
+import org.springframework.hateoas.ExposesResourceFor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.net.URI;
+
+@RepositoryRestController
+@RequiredArgsConstructor
+@ExposesResourceFor(ExportEntity.class)
+public class ExportEntityController {
+    private final ExportEntityService exportEntityService;
+    private final ExportEntityRepository exportEntityRepository;
+    private final PagedResourcesAssembler pagedAssembler;
+
+    @GetMapping(path = Links.EXPORT_JOB_URL + "/{id}/entities")
+    ResponseEntity<?> getExportJobEntities(@PathVariable("id") ExportJob exportJob,
+                                           @RequestParam(name = "status", required = false) ExportState exportState,
+                                           Pageable pageable,
+                                           PersistentEntityResourceAssembler assembler) {
+        Page<ExportEntity> entityPage;
+        if (exportState == null) {
```

Comment:
Could use the Spring Data query-by-example here instead, though I'm not a fan of how it doesn't allow us to enforce non-null constraints